metis-indexing/metis-indexing-client/src/test/java/eu/europeana/indexing/client/test/IndexingWorkerTestWithMain.java

```diff
@@ -0,0 +1,111 @@
+package eu.europeana.indexing.client.test;
+
+import eu.europeana.indexing.client.IndexingClient;
+import eu.europeana.indexing.client.IndexingWorker;
+import eu.europeana.metis.exception.IndexingException;
+
+public class IndexingWorkerTestWithMain {

```

Comment:
This is not a proper test class (it has a main method).

Reply:
As you've seen earlier with enrichment/dereferencing, the "Test with Mains" classes are used to run integration-type tests and test overall flows. They are not unit tests.